code/mob/living/critter/small_animal.dm

```diff
@@ -141,6 +141,8 @@ ABSTRACT_TYPE(/mob/living/critter/small_animal)
 	ai_type = /datum/aiHolder/mouse
 	is_npc = TRUE
 	ai_retaliates = TRUE
+	ai_retaliate_patience = 0
+	ai_retaliate_persistence = 0
```

Comment:
will probably want a comment here talking about what this value does, seems to be slightly confusing w/ the implementation.